services/workflows-service/src/workflow/schemas/zod-schemas.ts

```diff
@@ -56,6 +56,10 @@ export const ConfigSchema = z
       .optional()
       .describe('Main representative of the company'),
     customerName: z.string().optional().describe('The customer (tenant) display name'),
+    isManualCreation: z
```

Comment:
instead a boolean why dont we just add a createdBy field?

Reply:
Good point, we already have createdBy - it will help to add the agent who created it there for auditing.
But, we wont know its a user id without marking its a manual creation. @liorzam wdyt?

Reply:
@liorzam 
`isManualCreation` - Is set not after creation, its parameter which indicates if manual creation is available within backoffice.

Though assigning id of agent to`createBy` on manual creation is a good point.